contracts/toCKB-typescript/src/utils/types/toCKB_cell.rs

```diff
@@ -26,7 +27,7 @@ impl ToCKBCellDataView {
         let status = ToCKBStatus::from_int(data_reader.status().to_entity().into())?;
         let lot_size = data_reader.lot_size().as_slice()[0];
         let user_lockscript = data_reader.user_lockscript().to_entity().as_bytes();
-        let x_lock_address = data_reader.x_lock_address().to_entity().as_bytes();
+        let x_lock_address = data_reader.x_lock_address().to_entity().raw_data();
```

Comment:
rebase master would be fine, i have pushed a commit today: unpack `x_lock_address` and `x_unlock_address`